pkg/alertmanager/alertmanager.go

```diff
@@ -90,6 +90,7 @@ type Config struct {
 	ExternalURL                       *url.URL
 	Limits                            Limits
 	Features                          featurecontrol.Flagger
+	Templates                         []string
```

Comment:
I think this belongs in the Alertmanager struct, as nothing else in Config gets updated by ApplyConfig.

We'll also need a lock around it, because ApplyConfig and the TestReceiversHandler can be called concurrently from different Goroutines,